pkg/types/baremetal/metadata.go

```diff
@@ -0,0 +1,9 @@
+package baremetal
+
+// Metadata contains baremetal metadata (e.g. for uninstalling the cluster).
+type Metadata struct {
+	LibvirtURI string `json:"libvirt_uri"`
+	IronicURI  string `json:"ironic_uri"`
+	APIVIP     string `json:"api_vip"`
```

Comment:
why are the VIPs required in metadata??

Reply:
https://github.com/openshift-metal3/kni-installer/pull/141